contextual_orchestrator/batch_routing.py

```
@@ -524,6 +538,130 @@ def retrieve(self, job: BatchJob) -> List[EmbeddingBatchResultItem]:
+    def _post(self, model: str, inputs: List[str]) -> Dict[str, Any]:
+        from .credentials import get_credential
+
+        request = Request(
+            f"{self._base_url}/embeddings",
+            data=json.dumps({"model": model, "input": inputs}).encode("utf-8"),
+            headers={
+                "authorization": f"Bearer {get_credential(self._credential_key) or ''}",
+                "content-type": "application/json",
+            },
+            method="POST",
+        )
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)